Backpack/Rating/Classes/BPKRatingBubble.m

```diff
@@ -0,0 +1,169 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKRatingBubble.h"
+
+#import <Backpack/Color.h>
+#import <Backpack/Common.h>
+#import <Backpack/Font.h>
+#import <Backpack/Label.h>
+#import <Backpack/Spacing.h>
+
+NS_ASSUME_NONNULL_BEGIN
+@interface BPKRatingBubble ()
+@property(nonatomic) BPKLabel *ratingBubbleLabel;
+@end
+
+@implementation BPKRatingBubble
+
+- (instancetype)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+
+    if (self) {
+        [self setUp];
+    }
+
+    return self;
+}
+
+- (nullable instancetype)initWithCoder:(NSCoder *)aDecoder {
+    self = [super initWithCoder:aDecoder];
+
+    if (self) {
+        [self setUp];
+    }
+
+    return self;
+}
+
+- (void)setUp {
+    _lowRatingColor = BPKColor.red500;
+    _mediumRatingColor = BPKColor.yellow500;
+    _highRatingColor = BPKColor.green500;
```

Comment:
Are we using ivars here form some particular reason?

Reply:
Woops!